benchmarks/actors-reactors/src/main/scala/org/renaissance/actors/Reactors.scala

```diff
@@ -273,6 +273,7 @@ object Fibonacci {
     system.spawn(Reactor[Int] { self =>
       self.main.events.onEvent { x =>
         done.success(x)
+        self.main.seal()
```

Comment:
Good catch.

Reply:
Good catch indeed!

Just one thing &mdash; in other benchmarks we seal the channel before completing the future. I don't know if that would be a problem here, it just caught my eye.

Reply:
Feel free to regularize as you see fit :-) Alex is the best authority on this, my feeling is sealing of the main channel will only have an effect after the event handler returns so it should not matter ...

Reply:
Your ordering might even be the better one, if I consider channel as a resource that should be cleaned up after the primary activity (future completion) of the handler. I have a stash of various cleanups in `reactors` from April/May, when I was looking for the race, so I'll do something about it when I get to making a PR out of it :-)

Feel free to merge.

Reply:
I think that the order inside `onEvent` should not matter much (but if you put it before, you will potentially very slightly decrease the number of events that are pushed onto that channel's queue, because it's closed sooner).

Reply:
(IIRC as soon as `seal` is called, all subsequent events delivered to that channel are dropped)

Reply:
I think all the workloads only send events when the recipient has not yet closed their channel (the communication pattern is such that everybody either only reacts or knows when to stop), so there should really be no difference.